src/main/java/com/tourguide/myclassjournal/TourGuideController.java

```diff
@@ -0,0 +1,15 @@
+package com.tourguide.myclassjournal;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class TourGuideController {
+
+    @GetMapping("/")
+    public String index() {
+        return "start";
+    }
+
+}
```

Comment:
As I mentioned with the TourGuideApplication, the controller had also been deleted with no replacements. So I made a simple TourGuideController.
Source: [https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3](https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3)